cmd/db.go

```diff
@@ -1,9 +1,16 @@
 package cmd
 
 import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"code.cloudfoundry.org/bytefmt"
```

Comment:
We try and follow the principle of a little copy and paste is better than a little dependency = Could we copy the files out of these repos (ensure the correct license header & copyright owner is listed) and then make them available as :

flanksource/commons/text.HumanSize and text.HumanDuration ?